README.md

```diff
@@ -61,6 +61,24 @@ In front-end, create a file named '.env' with the following contents:
 
 Replace [your API key] with your actual API key. Save the file and close it.
 
+For example, if your API key is 123456, the file would read:
+
+> REACT_APP_MOCKAROO_KEY=123456
+
+Then, run:
+
+> npm start
+
+To start the back end of the project, navigate to the back-end folder and run the following command in the command line:
+
+> npm install
+
+In back-end, create a file named '.env' with the following contents:
+
+> REACT_APP_MOCKAROO_KEY=[your API key]
```

Comment:
Just my two cents, but I think we could rename this to something else other than `REACT_APP_MOCKAROO_KEY`? Since we don't need the `REACT_APP` appended to it, I think it'd be better to be named just `MOCKAROO_KEY`.

Reply:
Makes sense, will change